sql/mysqld.cc

```diff
@@ -6639,6 +6639,9 @@ int mysqld_main(int argc, char **argv)
     unireg_abort(1);
 
 #ifdef WITH_WSREP /* WSREP AFTER SE */
+  // Initialize wsrep_provider_set before anything else wsrep related
+  wsrep_provider_set = wsrep_provider != NULL && strcmp(wsrep_provider, WSREP_NONE) != 0;
```

Comment:
I think here wsrep_provider is always NULL (not initialized yet, so we probably can commit this at all (as wsrep_provier_set is already set to false)

Reply:
No, at this point sysvars are already initialized, this is where we set its initial value to true when it is specified at the command line / as a parameter.